users/updates/components/86FkFVV4TGRg.php

```diff
@@ -24,7 +24,7 @@
   $errors[] = "Unable to update group_menu int columns to 11 and unsigned, Error: ".$error;
 }
 
-$db->query("ALTER TABLE users MODIFY COLUMN logins integer NOT NULL");
+$db->query("ALTER TABLE users ADD COLUMN logins integer NOT NULL");
```

Comment:
Original was `MODIFY`, why the `ADD`?

Reply:
MODIFY gives syntax error in postgresql so i used ADD. I think i can also use ALTER keyword again but ADD works too I tested it